src/components/CardHome.js

```diff
@@ -13,12 +14,15 @@ function CardHome({exercise}) {
 				<a>wirkung</a>
 			</Link>
 			<button
+				disabled={
+					routine.filter(routine => routine.id === exercise.id).length > 0 ? true : false
+				}
```

Comment:
The good news: `length` is a truthy value, so you do not have to check whether it is true or false. If the exercise is included in the routine, the length will be 1 e.g. truthy, so the whole expression will resolve to `true`. 
```suggestion
					routine.filter(routine => routine.id === exercise.id).length}
```

Reply:
Ok erledigt .